packages/discord.js/src/structures/Webhook.js

```diff
@@ -4,6 +4,7 @@ const { makeURLSearchParams } = require('@discordjs/rest');
 const { DiscordSnowflake } = require('@sapphire/snowflake');
 const { Routes, WebhookType } = require('discord-api-types/v10');
 const MessagePayload = require('./MessagePayload');
+const { User } = require('..');
```

Comment:
```suggestion
const User = require('./User');
```

Reply:
damn, thought I got it to do all of them correctly 😢